resources/views/livewire/public_home/today_financial_reports.blade.php

```diff
@@ -0,0 +1,57 @@
+<div class="page-header">
+    <h3 class="page-title">{{ __('report.summary_weekly') }}</h3>
+    <div class="page-options d-flex">
+        <a class="btn btn-sm btn-success" href="{{ route('public_reports.in_weeks') }}"
+            role="button">{{ __('app.show') }}</a>
+    </div>
+</div>
+<div class="card">
+    <table class="table table-sm mb-0">
+        <tbody>
+            <tr>
+                <td class="col-4">
+                    <p class="m-0 px-2">{{ 'Saldo per ' .now()->startOfWeek()->isoFormat('dddd, D MMMM Y') }}</p>
```

Comment:
Bagian viewnya kita kasih element span dengan attribute id, sesuai yang ditesting
```suggestion
                    <span id="start_week_label">{{ 'Saldo per '.now()->startOfWeek()->isoFormat('dddd, D MMMM Y') }}</span>
```